fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -207,11 +208,13 @@ public LaoDetailViewModel(
       @NonNull Application application,
       LAORepository laoRepository,
       Gson gson,
+      Wallet wallet, // FIXME Use KeyManager instead
```

Comment:
FIXME... when ? :-)

Reply:
I don't want to change everything at once, so I add FIXME to do it on later commit.

I plan to fix them all before the end fo the feature.
You can be sure that it won't go on master.